### PR TITLE
Titlebar buttons update

### PR DESCRIPTION
**Pull request by QNetITQ**
Aligned the buttons vertically on the tab bar

### CODE REVIEW

**Comment by black7375** [comment]
If it is good to match vertial align, but it shouldn't remove the feature.

**Comment by QNetITQ** [comment]
What do you mean by "feature"?

File: userChrome.css [hunk 1]
```diff
@@ -850,45 +828,6 @@
     margin-inline-end: 1px !important;
   }
 
-  /* New tab button - Looks like tab ******************************************/
```
Comment (black7375) [inline userChrome.css:853]: Newtab related feature have been removed.

Comment (black7375) [inline userChrome.css:853]: Newtab related feature have been removed.

**Comment by QNetITQ** [comment]
Is this really so necessary? It may be easier to make a button look similar to other buttons to match the overall styling. Or is this moment important to you?

**Comment by black7375** [comment]
You can see the background introduced in #95 #96
I don't want to remove the current new tab button. It's intuitive and natural.

**Comment by QNetITQ** [comment]
Here's what I found out. 
изображение
This directive causes distortion of the new tab button, and also increases the height of the tab bar by 2px in windowed mode. This must be fixed first, otherwise the buttons will not be put in order.

**Comment by black7375** [comment]
**Current Ver**
image

**Your Ver**
image

Current version's alignment seems to be better.

---

> изображение
These directives distort the new tab button and also increase the height of the tab bar by 2 pixels in windowed mode. It is advisable to delete them, otherwise the buttons will not be put in order. Only after that it will be possible to move on.

I'll find a better way.

**Comment by QNetITQ** [comment]
We decided that you would not apply my version, as it seemed to me. This amendment is for your style. Without it, your new tab button is flattened.

**Comment by black7375** [comment]
I appreciate your contribution efforts, and we will refer to them for further fine-tuning.

**Comment by QNetITQ** [comment]
My business is to offer, yours is to refuse or agree. But keep in mind, I'm using my own style. I tried to fix yours only because of the great support of the community. But not the fact that I will do it in the future

**Pull request closed without merging**